test/test_utils.py

```diff
@@ -781,3 +785,163 @@ def test_get_org_environment_owner(self):
         self.mock_uep.getOwner = Mock(return_value=self.MOCK_ORG_ENVIRONMENT_OWNER)
         ret = is_simple_content_access(uep=self.mock_uep, identity=self.identity)
         self.assertTrue(ret)
+
+
+PROCESS_STATUS_FILE = \
+"""Name:\t{name}
+Umask:	0000
+State:	S (sleeping)
+Tgid:	1
+Ngid:	0
+Pid:	1
+PPid:	0
+TracerPid:	0
+Uid:	0	0	0	0
+Gid:	0	0	0	0
+FDSize:	512
+Groups:
+NStgid:	1
+NSpid:	1
+NSpgid:	1
+NSsid:	1
+VmPeak:	  237396 kB
+VmSize:	  180056 kB
+VmLck:	       0 kB
+VmPin:	       0 kB
+VmHWM:	   16216 kB
+VmRSS:	    7800 kB
+RssAnon:	    3768 kB
+RssFile:	    4032 kB
+RssShmem:	       0 kB
+VmData:	   29584 kB
+VmStk:	     132 kB
+VmExe:	     912 kB
+VmLib:	    9380 kB
+VmPTE:	     108 kB
+VmSwap:	    1564 kB
+HugetlbPages:	       0 kB
+CoreDumping:	0
+THP_enabled:	1
+Threads:	1
+SigQ:	1/63421
+SigPnd:	0000000000000000
+ShdPnd:	0000000000000000
+SigBlk:	7be3c0fe28014a03
+SigIgn:	0000000000001000
+SigCgt:	00000001800004ec
+CapInh:	0000000000000000
+CapPrm:	0000003fffffffff
+CapEff:	0000003fffffffff
+CapBnd:	0000003fffffffff
+CapAmb:	0000000000000000
+NoNewPrivs:	0
+Seccomp:	0
+Speculation_Store_Bypass:	thread vulnerable
+Cpus_allowed:	f
+Cpus_allowed_list:	0-3
+Mems_allowed:	00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000001
+Mems_allowed_list:	0
+voluntary_ctxt_switches:	4002321
+nonvoluntary_ctxt_switches:	16572
+"""
```

Comment:
I would add here few cases for status file containing:

- First line empty
- First line containing only 'Name:'
- Empty file
- Maybe something else